waspc/data/Cli/templates/basic/package.json

```diff
@@ -5,6 +5,8 @@
     "react": "^18.2.0"
   },
   "devDependencies": {
+    "typescript": "^5.1.0",
+    "vite": "^4.3.9",
```

Comment:
The SDK needs these two for building. Also, our framework code relies on them existing.

Therefore, I concluded this is the best place to specify them as dependencies.